translation/en/RedirectedResources/assets/abdata/adv/scenario/c11/20/1100/translation.txt

```diff
@@ -29,7 +29,7 @@
 「うん……俺もこの瞬間、待ち遠しかったよ」=“Yeah... I've been waiting for this moment, for so long too.”
 「さあ、手を出して…………」=“Come now, hold out your hand...”
 「ふふふ、素敵……今日のあなたのためにデザインされたみたいな指輪ね」=“Fufufu, it's lovely... feels like it was designed just for your hand.”
-「[H名]の方だって、最高に似合ってるよ。綺麗過ぎて、まともに見られないぐらいだもの」=“Same goes for you [H名], it looks absolutely fantastic on you. You look so beautiful, it's almost hard to stare directly at you.”
+「[H名]の方だって、最高に似合ってるよ。綺麗過ぎて、まともに見られないぐらいだもの」=“Same goes for yourself, it looks absolutely fantastic on you. You look so beautiful, it's almost hard to stare directly at you.”
```

Comment:
Okay with dropping the name, but "Same goes for yourself" is awkward.

```suggestion
「[H名]の方だって、最高に似合ってるよ。綺麗過ぎて、まともに見られないぐらいだもの」=“Same goes for you, it looks absolutely fantastic on you. You look so beautiful, it's almost hard to stare directly at you.”
```

Of course that means 4 `you`s in 2 sentences.

Maybe:
```suggestion
「[H名]の方だって、最高に似合ってるよ。綺麗過ぎて、まともに見られないぐらいだもの」=“Same, it looks absolutely fantastic on you. You look so beautiful, it's almost blinding.”
```